edx_toggles/tests/test_setting_toggles.py

```diff
@@ -78,3 +79,62 @@ def test_deleted_instances_are_not_listed(self):
         toggles.SettingToggle("NAME1", default=False, module_name="module1")
         instances = toggles.SettingToggle.get_instances()
         self.assertEqual([], instances)
+
+
+class NestedToggleExtractionTests(TestCase):
+    """
+    Tests for deeply nested toggle extraction from report.py
+    """
+
+    def test_level_1_nesting(self):
+        """
+        Test that Level 1 nested boolean values are correctly extracted.
+
+        Level 1: CONFIG['simple_toggle']
+        """
+        # Level 1 nesting: CONFIG['simple_toggle']
+        nested_config = {
+            'simple_toggle': True,
+        }
+
+        settings_dict = {}
+        _add_setting(settings_dict, nested_config, 'CONFIG')
+
+        # Test Level 1 nesting
+        level1_toggle = "CONFIG['simple_toggle']"
```

Comment:
Nit: I like using `expected` sometimes to make it more clear that this isn't being used for setup, but just as an expected value. This is just an opinion, so no change is necessary if you don't like the suggestion.
```suggestion
        expected_level1_toggle = "CONFIG['simple_toggle']"
```